contracts/oracle/integration/test-oracle.ava.js

```diff
@@ -65,3 +65,24 @@ test("Should pass, oracle is an issuer", async (t) => {
   const is_used_identity = await claimer.call(oracle_contract, "is_used_identity", { 'external_id': external_id}, { gas: 200000000000000 });
   t.true(is_used_identity);
 });
+
+test("Should pass, mint sbt token and revoke (burn)", async (t) => {
+  const { registry_contract, oracle_contract, admin, claimer } = t.context.accounts;
+  const add_issuer_result = await admin.call(registry_contract, "admin_add_sbt_issuer", {'issuer': oracle_contract.accountId});
+  t.is(add_issuer_result, true);
+  let supply_by_issuer = await oracle_contract.call(registry_contract, "sbt_supply", {'issuer': oracle_contract.accountId}, { gas: 200000000000000});
```

Comment:
we are doing a query here, so:
1. let's use `view` instead of `call`
2. gas is not needed for view methods
Probably need to update other places in this file when we do "view" calls.

Reply:
DONE